src/jwt.rs

```diff
@@ -738,4 +741,38 @@ mod test {
         assert_eq!(claims, decoded);
         assert_eq!(claims.metadata.unwrap().valid_signers.unwrap().len(), 1);
     }
+
+    #[test]
+    fn encode_decode_bad_token() {
+        let kp = KeyPair::new_account();
+        let claims = Claims {
+            metadata: Some(Actor::new(
+                "test".to_string(),
+                Some(vec![MESSAGING.to_string(), KEY_VALUE.to_string()]),
+                Some(vec![]),
+                false,
+                Some(1),
+                Some("".to_string()),
+            )),
+            expires: None,
+            id: nuid::next(),
+            issued_at: 0,
+            issuer: kp.public_key(),
+            subject: "test.wasm".to_string(),
+            not_before: None,
+        };
+
+        let encoded_nosep = claims.encode(&kp).unwrap().replace(".", "");
+
+        let decoded = Claims::<Account>::decode(&encoded_nosep);
+        assert!(decoded.is_err());
+        if let Err(e) = decoded {
+            match e.kind() {
+                ErrorKind::Token(s) => assert_eq!(s, &encoded_nosep),
```

Comment:
Same as above, let's not use the full token as the error message but instead something someone can use to troubleshoot.